src/display/font_loader.js

```diff
@@ -25,28 +25,35 @@ import {
 } from "../shared/util.js";
 
 class BaseFontLoader {
-  constructor({ docId, onUnsupportedFeature }) {
+  constructor({
+    docId,
+    onUnsupportedFeature,
+    ownerDocumentContext = globalThis.document,
+  }) {
     if (this.constructor === BaseFontLoader) {
       unreachable("Cannot initialize BaseFontLoader.");
     }
     this.docId = docId;
     this._onUnsupportedFeature = onUnsupportedFeature;
+    this._ownerDocumentContext = ownerDocumentContext;
```

Comment:
Why not simply `this._document = ownerDocument;` instead, to reduce the verbosity in the rest of the code?